pkgs/development/python-modules/macropy/default.nix

```diff
@@ -0,0 +1,37 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, python
+, isPy27
+, pinqSupport ? true, sqlalchemy
```

Comment:
*comment:* I don't think we should pull in extras by default unless they were in `install_requires`.
However this is done to the packagers discretion everywhere already, but it does have some drawbacks.

Wish there was a better way to declare optional dependencies for python in nixpkgs.

Reply:
Fixed.